### DISCUSSION TITLE
Copilot.vim to be fully featured any time soon?

### DISCUSSION THREAD

**Discussion by ChrisThackrey** [community/community#9487] [Copilot Conversations] [answered]
Neovim plugin is great, could really use the next/prev suggestion feature exposed as a callable, as well as the generate function.

Wondering if we could get those implementations bumped higher on the roadmap?

Thanks, awesome job!

**Comment by Lucklyric** [comment, marked as answer]
Hi @ChrisThackrey , feel free to check this fork as a temporary solution. https://github.com/Lucklyric/copilot.vim,
